cron-jobs/egov-weekly-impact-notifier/src/main/resources/velocity/weeklyimpactflasher.vm

```diff
@@ -0,0 +1,182 @@
+<html>
+	<br>Hi All,</br>
+</br>
+<br>DIGIT is starting to be adopted across the country rapidly through NUS. While this adoption takes place its important for all of us to be aware the extent of its adoption .
```

Comment:
Can we simplify this? For instance, table rows and columns should ideally be generated by looping through a collection.

Reply:
Yes, but need more time to implement, we can send the change in an intermediate release later.